CONTRIBUTING.md

```diff
@@ -82,22 +82,49 @@ explanation about how something works.**
 It is important to create documentation that follows similar guidelines. This allows documentation
 to be clear and concise while allowing users to easily find necessary information.
 
-These are general style guidelines that can help create clearer documentation:
+### Substrate-Specific Style
+
+This section is for terminology specifically related to Substrate.
+
+- **WebAssembly:** It's "WebAssembly" or "Wasm".
+
+- **FRAME, Pallets, and Modules:** A module is a generic term for a piece of software that expresses
+  domain-specific logic. A pallet is a special type of module that is meant to operate in the
+  context of FRAME. FRAME is a system that provides the abstractions necessary for building a
+  runtime on the Substrate client. If you don't know if you should call something a module or a
+  pallet, check its `Cargo.toml` file for its name; pallets have a `pallet-*` prefix.
```

Comment:
This is true of the pallets that live in the main Substrate repo, but will not be true in general. For example see the [ORML](https://github.com/open-web3-stack/open-runtime-module-library/blob/master/auction/Cargo.toml) or the [Recipes](https://github.com/substrate-developer-hub/recipes/blob/master/pallets/compounding-interest/Cargo.toml#L2).

I'd be happy to help normalize this convention if you like, but even still it won't be true across the board.